lib/taskana-core-test/src/test/java/acceptance/classification/update/UpdateClassificationAccTest.java

```diff
@@ -0,0 +1,834 @@
+                  classification.getName()));
+      assertThatThrownBy(() -> classificationService.updateClassification(classification))
+          .usingRecursiveComparison()
+          .isEqualTo(expectedException);
+    }
+
+    @WithAccessId(user = "businessadmin")
+    @Test
+    void should_NotThrowException_When_UpdatingClassificationWithEmptyServiceLevel()
+        throws Exception {
+      Classification classification =
+          defaultTestClassification().serviceLevel("P1D").buildAndStore(classificationService);
+      classification.setServiceLevel("");
+      assertThatCode(() -> classificationService.updateClassification(classification))
+          .doesNotThrowAnyException();
+    }
+  }
+
+  @TestInstance(Lifecycle.PER_CLASS)
+  @Nested
+  class UpdateClassificationCategoryTest {
+    Classification classification;
+    Task task;
+    Instant createdBefore;
+    Instant modifiedBefore;
+
+    /**
+     * User Role is admin here since we create and get Tasks.
+     *
+     * @throws Exception for errors in the building, updating or reading process of entities.
+     */
+    @WithAccessId(user = "businessadmin")
+    @BeforeEach
+    void createClassificationAndTask() throws Exception {
+      classification =
+          defaultTestClassification()
+              .category("MANUAL")
+              .type("TASK")
+              .buildAndStore(classificationService);
+      createdBefore = classification.getCreated();
+      modifiedBefore = classification.getModified();
+      String taskId = createTaskWithExistingClassification(classification.asSummary());
+      task = taskService.getTask(taskId);
+    }
+
+    /**
+     * User Role is admin here since we create and get Tasks.
+     *
+     * @throws Exception for errors in the building, updating or reading process of entities.
+     */
+    @WithAccessId(user = "businessadmin")
+    @Test
+    void should_UpdateClassification_When_UpdatingClassificationCategory() throws Exception {
+      classification.setCategory("PROCESS");
+      classificationService.updateClassification(classification);
+      final Task updatedTask = taskService.getTask(task.getId());
+
+      TaskImpl expectedUpdatedTask = (TaskImpl) task.copy();
+      expectedUpdatedTask.setId(task.getId());
+      expectedUpdatedTask.setClassificationCategory("PROCESS");
+      expectedUpdatedTask.setClassificationSummary(
+          classificationService.getClassification(classification.getId()).asSummary());
+      expectedUpdatedTask.setExternalId(task.getExternalId());
+      assertThat(expectedUpdatedTask)
+          .usingRecursiveComparison()
+          .ignoringFields("modified")
+          .isEqualTo(updatedTask);
+      assertThat(expectedUpdatedTask.getModified()).isAfterOrEqualTo(modifiedBefore);
+    }
+
+    @WithAccessId(user = "businessadmin")
+    @Test
+    void shouldUpdateTask_When_UpdatingClassificationCategory() throws Exception {
```

Comment:
There should be an underscore after `should`. Where do we check update of a Task in this test?